pkg/panels/kubernetes/cluster.go

```diff
@@ -0,0 +1,484 @@
+					Header: "Pods",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #2",
+					Header: "Workloads",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #3",
+					Header: "CPU Usage",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #4",
+					Header: "CPU Requests",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #5",
+					Header: "CPU Requests %",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #6",
+					Header: "CPU Limits",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name:   "value #7",
+					Header: "CPU Limits %",
+					Align:  tablePanel.RightAlign,
+				},
+				{
+					Name: "timestamp",
+					Hide: true,
+				},
+			}),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(kube_pod_owner{"+GetKubeStateMetricsMatcher()+", cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"count(avg(namespace_workload_pod:kube_pod_owner:relabel{cluster=\"$cluster\"}) by (workload, namespace)) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum("+GetNodeNSCPUSecondsRecordingRule()+"{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(namespace_cpu:kube_pod_container_resource_requests:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum("+GetNodeNSCPUSecondsRecordingRule()+"{cluster=\"$cluster\"}) by (namespace) / sum(namespace_cpu:kube_pod_container_resource_requests:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum(namespace_cpu:kube_pod_container_resource_limits:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+		panel.AddQuery(
+			query.PromQL(
+				promql.SetLabelMatchers(
+					"sum("+GetNodeNSCPUSecondsRecordingRule()+"{cluster=\"$cluster\"}) by (namespace) / sum(namespace_cpu:kube_pod_container_resource_limits:sum{cluster=\"$cluster\"}) by (namespace)",
+					labelMatchers,
+				),
+				dashboards.AddQueryDataSource(datasourceName),
+			),
+		),
+	)
+}
+
+func ClusterMemoryUsageQuota(datasourceName string, labelMatchers ...promql.LabelMatcher) panelgroup.Option {
+	return panelgroup.AddPanel("Memory Quota",
```

Comment:
Hi, just a small nit pic but we try to keep the naming of panels consistent with existing dashboard https://github.com/kubernetes-monitoring/kubernetes-mixin/blob/master/dashboards/resources/cluster.libsonnet#L254

Reply:
Ah yes, these are basically the same tables, so I was copying things over, will update